File: Beecrowd/conversaoTempo.java
import java.util.Scanner;

public class conversaoTempo {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);

    int segundo, minuto, hora;

    segundo = sc.nextInt();
    hora = segundo / 3600;
    segundo = segundo % 3600;

    minuto = segundo / 60;
    segundo = segundo % 60;

    System.out.printf("%d:%d:%d\n", hora, minuto, segundo);
    sc.close();
  }
}

File: Beecrowd/C/ConversaoTempo.c
#include <stdio.h>

int main() {
  int tempo, hora, minuto, segundo;
  scanf("%d", &tempo);

  hora = tempo / 3600;
  tempo = tempo % 3600;

  minuto = tempo / 60;
  tempo = tempo % 60;

  segundo = tempo;

  printf("%d:%d:%d\n", hora, minuto, segundo);

  return 0;
}
